lib/mongo_client.js

```diff
@@ -118,6 +118,17 @@ const closeOperation = require('./operations/mongo_client_ops').closeOperation;
  * @param {number} [options.minSize] If present, the connection pool will be initialized with minSize connections, and will never dip below minSize connections
  * @param {boolean} [options.useNewUrlParser=false] Determines whether or not to use the new url parser. Enables the new, spec-compliant, url parser shipped in the core driver. This url parser fixes a number of problems with the original parser, and aims to outright replace that parser in the near future.
  * @param {boolean} [options.useUnifiedTopology] Enables the new unified topology layer
+ * @param {object} [options.autoEncryption] Optionally enable client side auto encryption
+ * @param {MongoClient} [options.autoEncryption.keyVaultClient] A `MongoClient` used to fetch keys from a key vault
```

Comment:
q: do users have to pass a separate `MongoClient` in to their `MongoClient` to get encryption working?

Reply:
No. By default one is created internally for them, this simply let's them override that with their own separate client.